app/controllers/profiles_controller.rb

```diff
@@ -15,4 +15,10 @@ def show
     authorize profile
     render json: ProfileSerializer.new(profile)
   end
+
+  private
+
+  def resource
```

Comment:
I'm confused where we're using this private method.

Reply:
In `search.rb`, we use it to know which model to scope_search for